apps/taquito-test-dapp/vite.config.ts

```diff
@@ -29,6 +29,13 @@ export default ({ command }) => {
           }/index.js`
           // `../../node_modules/@airgap/beacon-dapp/dist/walletbeacon.dapp.min.js`
         ),
+        "@airgap/beacon-sdk": path.resolve(
+          path.resolve(),
+          // "./src/walletbeacon.dapp.min.js"
```

Comment:
are these comments needed? If not then it's probably better to clean up these artifacts